### PR TITLE
New us government agencies cheat sheet

### PR DESCRIPTION
**Pull request by aasoliz**
**What does your Instant Answer do?**
Displays acronyms for common US government agencies. 

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It's shortens the list of government agencies provided in the source. It only shows common agencies that people are more likely to be interested in.

**What is the data source for your Instant Answer? (Provide a link if possible)**
A Guide to U.S. Government Acronyms & Abbreviations
http://ucsd.libguides.com/c.php?g=90902&p=584912

**Why did you choose this data source?**
It has, if not all, most of the us government agencies.

**Are there any other alternative (better) data sources?**
Not that I know of.

**What are some example queries that trigger this Instant Answer?**
us government agencies, us government, us agencies

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Reference community

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
no

**Which existing Instant Answers will this one supercede/overlap with?**
none

**Are you having any problems? Do you need our help with anything?**
no

**Where did you hear about DuckDuckHack? (For first time contributors)**
While signing up for "Share it & Wear it" I clicked on the develop tab.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

 - Linux
     - [x] Firefox
     - [x] Google Chrome
```

-----
IA Page: https://duck.co/ia/view/us_government_agencies_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@aasoliz thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/json/us-government-agencies.json [hunk 1]
```diff
@@ -0,0 +1,308 @@
+{
+    "id": "us_government_agencies_cheat_sheet", 
+    "name": "Acronyms for US Government Agencies (Noncomprehensive)",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/us-government-agencies.json]: I think we can remove the "Noncomprehensive"

File: share/goodie/cheat_sheets/json/us-government-agencies.json [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    "description": "Common US Government Agencies and their Acronyms", 
+    "metadata": { 
+        "sourceName": "Common U.S. Government Abbreviations and Acronyms",
+        "sourceUrl": "http://www15.uta.fi/FAST/GC/usabacro.html"
+    },	
+    "aliases": [
+        "us government agencies", "us government", "us agencies"
+    ],
+    "template_type": "code",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/us-government-agencies.json]: I think we should use the "Reference" template. It was made for this kind of content specifically! Checkout https://duckduckgo.com/?q=harry+potter+spells+cheat+sheet&ia=cheatsheet

**Comment by moollaza** [comment]
@aasoliz thanks a lot for contributing, we really appreciate it! I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by aasoliz** [comment]
Thanks for a quick response! I changed the template to reference and removed the "noncomprehensive".

**Comment by moollaza** [comment]
@aasoliz this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@aasoliz congrats! It's live: https://duckduckgo.com/?q=US+Government+Agencies+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
By the way, we think you did a great job with this! Are you interested in helping others who build similar Instant Answers in the future?

If yes, we'll reach out and add you to a private list of mentors as well as send over some DDG merch as an additional thank you. Just email <open@duckduckgo.com> with the subject 'I want to help.' Please include your shipping address, phone-number (required for shipping outside USA) and t-shirt size. We'll reach out from there!

Otherwise, thank you for this amazing contribution. We hope you'll keep exploring and playing around with DuckDuckHack.

**Pull request merged**